apocalypse_sim/grid_map/map_layouts.py

```diff
@@ -36,6 +36,11 @@ def __init__(self, map_id, model):
             maps = [self.simple_incubation]
         elif os.environ["mode"] == "4":
             maps = [self.doorway_map]
+        elif os.environ["mode"] == "5":
```

Comment:
This mode can be removed as it was only used for reference when creating the poster